### PR TITLE
[PREVIEW] Add in Root Controller

### PR DESCRIPTION
**Pull request by tchow8**
# Description

Adds a Root Controller to EMCA so that Azure checks to '/' do not return a 404 and instead returns a 200 so we don't get any false positive errors.

### CODE REVIEW

File: src/test/java/uk/gov/hmcts/reform/emclient/controller/RootControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import uk.gov.hmcts.reform.emclient.application.EvidenceManagementClientApplication;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@RunWith(SpringRunner.class)
+@WebMvcTest(RootController.class)
+@ImportAutoConfiguration({RibbonAutoConfiguration.class,HttpMessageConvertersAutoConfiguration.class, FeignRibbonClientAutoConfiguration.class, FeignAutoConfiguration.class})
```
Comment (mthava) [inline src/test/java/uk/gov/hmcts/reform/emclient/controller/RootControllerTest.java]: Remove Ribbon, Context Config

Comment (tchow8) [reply src/test/java/uk/gov/hmcts/reform/emclient/controller/RootControllerTest.java]: I've changed all the other services according to your review comments but I can't seem to get this test working if I remove any of the parts, Ribbon or Context. Even with the same @SpringBootTest it seems to always be missing something if I remove anything.

Comment (mthava) [reply src/test/java/uk/gov/hmcts/reform/emclient/controller/RootControllerTest.java]: Clean up the annotation on top of the main class

Comment (mthava) [reply src/test/java/uk/gov/hmcts/reform/emclient/controller/RootControllerTest.java]: remove @EnableCircuitBreaker

Comment (mthava) [reply src/test/java/uk/gov/hmcts/reform/emclient/controller/RootControllerTest.java]: and @EnableRetry(proxyTargetClass=true)

**Comment by tchow8** [comment]
Closing due to age. Will raise a new one when required and set out by a ticket.

**Pull request closed without merging**